frontend/src/shared/components/lists/reviews-paginated-list.jsx

```diff
@@ -0,0 +1,114 @@
+import FormatQuoteIcon from "@mui/icons-material/FormatQuote";
+import {
+  Box,
+  Stack,
+  Typography,
+  Pagination,
+  Divider,
+  Avatar,
+  Rating,
+  CircularProgress,
+} from "@mui/material";
+
+function ReviewItem({ title, text, score, userName, createdAt }) {
+  return (
+    <Box>
+      <Stack spacing={2} direction={"row"} alignItems={"center"} sx={{ mb: 2 }}>
+        <Stack spacing={2} direction={"column"} alignItems={"center"}>
+          <Avatar>JD</Avatar>
```

Comment:
I think JD is not enough. Maybe you can generate the name using the first letter of the username.

Reply:
Utilize the first character of the username